File: Data Structures & Algorithms/anagram-groups/submission-0.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& strs) {
        for (auto e : strs) {

            std::string sorted = e;
            std::sort(e.begin(), e.end());
            std::cout << sorted;
        }
       return{}; 
    }
};

File: Data Structures & Algorithms/check-if-array-is-sorted-and-rotated/submission-0.py
class Solution:
    def check(self, nums: list[int]) -> bool:
        def rotated(xs: list[int], x: int) -> list[int]:
            n = len(xs)
            return [xs[(i + x) % n] for i in range(n)]
        xs = sorted(nums)
        for i in range(len(nums)):
            if rotated(xs, i) == nums:
                return True
        return False
